src/sentry/event_manager.py

```diff
@@ -358,9 +358,20 @@ def stringify(f):
         # See GH-3248
         msg_str = data.pop('message', None)
         if msg_str:
-            msg_if = data.setdefault('sentry.interfaces.Message', {'message': msg_str})
```

Comment:
I'm okay with this for now but I think we should probably leave a comment here why we do this and what we want to do going forward. There is a TODO item on the protocol iteration list for this logentry vs message thing.

Reply:
The comment is above that block, quite elaborate, actually.